source/includes/gcp/_routes.md

```diff
@@ -1 +1,101 @@
-### Routes
\ No newline at end of file
+### Routes
+
+A route is a paths that network traffic takes from an [instances](#gcp-instances) to other destinations. The destination can be inside a GCP network or outside.
```

Comment:
Should this be "a route is a path"?

Reply:
1c902795ba00f6ba1f4af8f97b9112267ef3e8a7